spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/MongoClientFactoryBean.java

```diff
@@ -291,7 +291,7 @@ protected MongoClientSettings computeClientSetting() {
 			applySettings(builder::retryWrites, computeSettingsValue(defaultSettings.getRetryWrites(),
 					mongoClientSettings.getRetryWrites(), connectionString.getRetryWritesValue()));
 			applySettings(builder::uuidRepresentation,
-					computeSettingsValue(null, mongoClientSettings.getUuidRepresentation(), UuidRepresentation.JAVA_LEGACY));
+					computeSettingsValue(null, mongoClientSettings.getUuidRepresentation(), UuidRepresentation.STANDARD));
```

Comment:
Why do we use here constants and not `connectionString.getUuidRepresentation()`?